go/libraries/doltcore/sqle/logictest/import.sql

```diff
@@ -0,0 +1,3 @@
+.mode csv
+.import nightly_mean.csv nightly_dolt_mean_results
+.import releases_mean.csv releases_dolt_mean_results
```

Comment:
importing csv to sqllite3 from the shell is not the business... using a file like this is very helpful.